src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -142,7 +142,8 @@ static <T> Blob constructBlobAndMetadata(
           chunkMetadataBuilder
               .setMajorVersion(Constants.PARQUET_MAJOR_VERSION)
               .setMinorVersion(Constants.PARQUET_MINOR_VERSION)
-              .setCreatedOn(0L)
+              // set createdOn in seconds
+              .setCreatedOn(System.currentTimeMillis() / 1000)
```

Comment:
Why do we need this? Blobs already have create time in its property and its cheaper to get it there than to query metadata

Reply:
For having consistent metadata across XP-based DML inserts into iceberg tables and SDK-based inserts into iceberg tables.